chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/endpoints/dto/ChaosMonkeyDisabledDto.java

```diff
@@ -1,10 +1,52 @@
 package de.codecentric.spring.boot.chaos.monkey.endpoints.dto;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.time.Duration;
 import java.time.ZonedDateTime;
 import lombok.Data;
 
 @Data
 public class ChaosMonkeyDisabledDto {
   private final String status = "Chaos Monkey is disabled";
-  private ZonedDateTime disabledAt = ZonedDateTime.now();
+  private final ZonedDateTime disabledAt = ZonedDateTime.now();
+  @JsonIgnore private Duration timeActive;
+  @JsonIgnore private Duration timeInActive;
+  private String timeActiveFormatted;
+  private String timeInActiveFormatted;
+
+  public ChaosMonkeyDisabledDto() {
+    this.timeActive = Duration.ZERO;
+  }
+
+  public ChaosMonkeyDisabledDto(ChaosMonkeyEnabledDto chaosMonkeyEnabledDto) {
+    this.timeActive = chaosMonkeyEnabledDto.getTimeActive();
+  }
```

Comment:
You could introduce an additional Constructor here which takes the timeactive as parameter.
Then you can call that from the other Constructors
eg. something like:
```java
  public ChaosMonkeyDisabledDto() {
    this(Duration.ZERO);
  }

  public ChaosMonkeyDisabledDto(ChaosMonkeyEnabledDto chaosMonkeyEnabledDto) {
    this(chaosMonkeyEnabledDto.getTimeActive());
  }
  public ChaosMonkeyDisabledDto(Duration timeActive){
    this.timeActive = timeActive;
    this.disabledAt = ZonedDateTime.now();
  }
```
There you can also set the disabledAt time stamp.